### PR TITLE
ACP2E-348: [Documentation] Add notice into the documentation that new product image is always assigned and visible in all Store Views

### PR DESCRIPTION
**Pull request by vkolesny**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

In scope of MC-41865 and MC-41693 issues investigation following Product Owner clarification was given:
https://jira.corp.magento.com/browse/MC-41693?focusedCommentId=1664621&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1664621

Following notice should be added in the Uploading Product Images document:
`New product images are **_always_** uploaded and visible in **_all_** Store Views. Even if they are uploaded not in the `All Store Views` scope. <br/>To hide any chosen product image from some specific `Store View`, you must switch to this `Store View`, click on the `Hide from Product Page` checkbox for this image, and click <span class="btn">Save</span>.`

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

Magento 2.4.x

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

Both Magento Open Source and Adobe Commerce

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

No

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

https://docs.magento.com/user-guide/catalog/product-image-upload.html

### CODE REVIEW

File: src/catalog/product-image-upload.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ If you plan to upload large images for viewing on the _Product Details_ page, yo
 
 1. To work with a specific store view, set the **Store View** chooser in the upper-left corner to the applicable view.
 
+{:.bs-callout-tip}
+New product images are **_always_** uploaded and visible in **_all_** Store Views. Even if they are uploaded not in the `All Store Views` scope. <br/>To hide any chosen product image from some specific `Store View`, you must switch to this `Store View`, click on the `Hide from Product Page` checkbox for this image, and click <span class="btn">Save</span>.
```
Comment (leslietilling) [inline src/catalog/product-image-upload.md:16]:
```suggestion
   {:.bs-callout-info}
   New product images are **_always_** uploaded and visible in **_all_** store views, even if the `All Store Views` scope is not used for upload. <br/><br/>To hide any product image from a specific store view, you must switch to that Store View , select the **Hide from Product Page** checkbox for the image, and click <span class="btn">Save</span>.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Editorial tweaks and some formatting

File: src/catalog/product-image-upload.md (hunk 1 above)
Comment (vkolesny) [reply src/catalog/product-image-upload.md]: Committed

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**